### ISSUE TITLE
problem in installer for nvda 2015.1 and later

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5160]
**Reported by JamaicanUser on 2015-06-18 01:21**
This problem occurs when trying to install a copy of NVDA 2015.1 and later, when browsing for where to place the copy of NVDA. when tabbing around, the focus lands on an 'explorer pane'. In prior versions of NVDA, this pane was easily navigable, and a new folder could easily be created. Now, the user cannot navigate this pane without object navigation. Could the problem be investigated/explained?

**Comment by bhavyashah** [comment]
Updates/inputs? @jcsteh @Qchristensen @leonardder

**Comment by Qchristensen** [comment]
Ok the original issue is correct.  In Windows 7, you can tab to that explorer pane when creating a portable copy.  As noted, none of this explorer pane reads with regular navigation.  The focus is actually on an Organise drop down button (pressing down arrow at that point correctly reads the options starting with cut, copy etc).  Instead of down arrow, if you press right arrow, you land on "New folder" button.  Press right arrow again and you land on another drop down list button thing which lets you change the item view list layout (extra large icons, list, details, tiles etc), and the last item in the explorer pane is help.  That is a fairly standard file open dialog.  This looks basically identical to the file save as dialog you get in Office.  In Word for instance, the bar is called "Command module toolbar" and is accessible, so it's curious it's not accessible here.  Maybe we do something specific in Office to access it?

On Windows 10, I get a different looking dialog titled "Browse for dialog" which then has a label "Select portable directory" (there's an extra space in between portable and directory btw), it then has a tree view  and under that an edit box with the current folder, a "Make new folder" button and OK and cancel buttons.  All of that is accessible.  For the sake of it, in Word, I get the same dialog I described in the previous paragraph with the command module toolbar which NVDA reads correctly.

**Comment by bhavyashah** [comment]
From @Qchristensen's https://github.com/nvaccess/nvda/issues/5160#issuecomment-319001356, the reported issue does not seem specific to NVDA's installer or portable copy creation dialog, but rather to the Explorer pane part of such dialogs on Windows 7. Since Quentin tested only with Windows 10, on Windows 8.1, the Create Portable NVDA dialog>Browse... button>Select Portable Directory dialog does not have any control named Explorer name in the Tab order. However, a control which does pose accessibility problems is 'header list name split button collapsed". Again, this control and its accessibility oddities are not specific to NVDA's Create Portable NVDA dialog but are applicable throughout Windows (perhaps 8.1 only?). At the very least, we may want to remove the installer label and morph this issue to encompass the Explorer pane (as well as header list name split button collapsed also?) control to better reflect what this ticket entails.

**Comment by leonardder** [comment]
I guess since this indeed doesn't seem to be related to NVDA and @Qchristensen mentioned some workarounds, it is safe to close this as a can't fix

**Comment by Adriani90** [comment]
@leonardder I agree, dialogs have changed alot since Windows XP but it is still accessible when you want to create a new folder. It is just another way to do it.
